src/utils/hints/accountHints.js

```diff
@@ -13,4 +13,9 @@ export const accountHints = {
   name: 'Name owned by this account.',
   expires: 'Keyblock height and estimated date and time when the name owned by the account expires.',
   pointsTo: 'The account that points to the name owned by the account.',
+  activityHash: 'Transaction hash of the transaction corresponding to the activity where the account was involved. Some of the activities are executed internally and therefore don\'t have a transaction hash.',
+  activityTime: 'Keyblock height and exact date and time when the activity has been executed by inclusion into a microblock.',
```

Comment:
here it's not easy for me to understand after reading it. Could you try something that would explain it for dummies? .)

Reply:
Same as above. It's copied from the hint above.